.github/workflows/publish_charm_arm64.yaml

```diff
@@ -0,0 +1,13 @@
+name: Publish to edge arm64
+
+on:
+  push:
+    branches:
+      - main
+      - track/*
+
+jobs:
```

Comment:
Don't you need to specify the track?

Reply:
The publish action is equal to the default for `amd64` (except that it uses the artifact from the arm64 integration test), by default it uses the `latest/stable`.